### PR TITLE
fix #3255

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qp_qiime2/qp_qiime2.py [hunk 1]
```diff
@@ -351,6 +369,10 @@ def call_qiime2(qclient, job_id, parameters, out_dir):
                     key_value).view(qiime2.Metadata)
             else:
                 q2inputs[key] = ('', '')
+        elif k in ('The set of backbone sequences in Greengenes2'):
```
Comment (charles-cowart) [inline qp_qiime2/qp_qiime2.py:372]: Are parentheses needed? This string looks like it's all on one line.

**Review by charles-cowart — CHANGES_REQUESTED** [review]
Everything looks good, just one question.

File: qp_qiime2/qp_qiime2.py (hunk 1 above)
Comment (antgonza) [reply qp_qiime2/qp_qiime2.py:372]: Yes if we want to do in to an exact string, another option is to use ==.

Comment (charles-cowart) [reply qp_qiime2/qp_qiime2.py:372]: ok

**Review by charles-cowart — APPROVED** [review]

**Pull request merged**